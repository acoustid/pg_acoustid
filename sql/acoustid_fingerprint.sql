SELECT acoustid_fingerprint(1, '{100,200,300}'::int4[]);
