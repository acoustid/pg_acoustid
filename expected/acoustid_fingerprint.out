SELECT acoustid_fingerprint(1, '{100,200,300}'::int4[]);
 acoustid_fingerprint 
----------------------
 {100,200,300}
(1 row)

